hardware_layer/hardware_arduino/include/arduino_thrust.h

```diff
@@ -0,0 +1,25 @@
+#ifndef arduino_thrust_h
+#define arduino_thrust_h
+
+using namespace std;
+
+class arduino_thrust
+{
+    public:
+    
+        arduino_thrust();
+    
+        //sets the direction and pwm pins for the thrusters and initiate them as outputs
+        void setPins(int A1, int D1, int D2);
+    
+        //applies analogwrite and digitalwrite functions in arduino
+        void ON(int pwm, bool Dir1, bool Dir2);
```

Comment:
DIR1 and DIR2 is calculated using the pwm sign so you don't need to take that as inputs

Reply:
Done